### PR TITLE
[User roles] Hide ACH confirmation letter link for non-managers

### PR DESCRIPTION
**Pull request by malted**
Before
image

After
image

### CODE REVIEW

File: app/views/hcb_codes/_ach_transfer.html.erb [hunk 1]
```diff
@@ -96,9 +96,11 @@
         <span>
           <%= @hcb_code.ach_transfer.status_text_long %>
           <% if @hcb_code.ach_transfer.deposited? %>
-            <span class="muted">
-              <%= link_to "(#{ organizer_signed_in? ? 'view confirmation letter' : 'sign in to view' })", ach_transfer_confirmation_path(@hcb_code.ach_transfer, format: :pdf), target: "_blank" %>
-            </span>
+            <% if policy(@event).new_transfer? %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_ach_transfer.html.erb]: combine with existing `if` above!

Comment (garyhtou) [inline app/views/hcb_codes/_ach_transfer.html.erb]: We should probably use a different policy for this

Comment (malted) [reply app/views/hcb_codes/_ach_transfer.html.erb]: Done!

Comment (malted) [reply app/views/hcb_codes/_ach_transfer.html.erb]: Done!

**Review by garyhtou — APPROVED** [review]
Great!

**Pull request merged**